app/assets/stylesheets/header.scss

```diff
@@ -1,8 +1,16 @@
 .header{
-  border-bottom:solid 2px #5DC9C9;
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  border-bottom:  2px solid #dcdcdc;
+  border-radius:  0 0 5px 5px;
+  box-shadow: 0 1px 1px #cecdcd inset;
+  background-image: linear-gradient(to bottom, #fafafa, #eaeaea);
 }
 .header-part{
-  margin: 1.5em 0;
+  margin: 0;
+  padding: 20px 0 0 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `20px 0 0` instead of `20px 0 0 0`